# Write a script that generates an exception.
# Handle this exception with a try/except block.
# For example:
#
list_ = ["hello world!"]



try:
    print(list_[1])
except:
    print("Out of range, silly!")
#
# This raises and exception that needs to be handled.





